.gitignore

```diff
@@ -73,8 +73,8 @@ outputs/*
 
 #ignore results
 input_files/bulk_example/
-input_files/*
-!input_files/*.txt
```

Comment:
Don't we still want to ignore the run output? If you did that to commit the .txt and .sv files in this merge request you can use something like this instead:
input_files/*
!input_files/**/.txt
!input_files/**/.sv

Reply:
Thanks for the review Mohamed.
The results don't go into input_files directory any more. They go into the path that the user specifies in the config files. That's a new option I added. But if someone doesn't provide a value for that option (which is how it is in some of the existing config files for example), then yes, the behavior like it used to be. That is, the outputs will go into input_files directory. I will update this based on this suggestion and make a commit in a bit.